_posts/2025-10-22-Buildah-version-v1.42.0.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Buildah version 1.42.0 Release Announcement
+layout: default
+author: tsweeney
+categories: [releases]
+
+tags: community, open source, buildah, hpc, opensource, containers, images, image
+---
+![buildah logo](https://buildah.io/images/buildah.png)
+
+# Buildah version 1.42.0 Release Announcement
+
+We're pleased to announce the release of [Buildah](https://github.com/containers/buildah) [version 1.42.0](https://github.com/containers/buildah/releases/tag/v1.42.0), which is now available from GitHub for any Linux distro.  We are shipping this release on Fedora 42 and Fedora 43.  Buildah will also be shipped on CentOS, OpenSUSE, and Ubuntu soon. In addition, container images will be available at https://quay.io/repository/buildah/stable and https://quay.io/repository/containers/buildah.
```

Comment:
```suggestion
We're pleased to announce the release of [Buildah](https://github.com/containers/buildah) [version 1.42.0](https://github.com/containers/buildah/releases/tag/v1.42.0), which is now available from GitHub for any Linux distro.  We are shipping this release on [Fedora 42](https://bodhi.fedoraproject.org/updates/FEDORA-2025-8a248ee4f4) and [Fedora 43](https://bodhi.fedoraproject.org/updates/FEDORA-2025-beb4c9a336).  Buildah will also be shipped on CentOS, OpenSUSE, and Ubuntu soon. In addition, container images will be available at https://quay.io/repository/buildah/stable and https://quay.io/repository/containers/buildah.
```

Reply:
added the links, thanks for the pointer!